inventory_refresh.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "activerecord", "~> 5.0"
+  spec.add_dependency "activerecord", ">=5.0", "< 6.1"
```

Comment:
Will this be loosened to drop the upper bound or "< 7.0" in a future PR?

Reply:
Yes